CODEOWNERS

```
@@ -0,0 +1,44 @@
+# Core
+/ghcide @pepeiborra
+/ghcide/session-loader @pepeiborra @fendor
+/hls-graph @pepeiborra
+/hls-plugin-api @jneira @berberman
+/hls-test-utils @jneira
+/hie-compat @wz1000
+
+# Plugins
+/plugins/hls-alternate-number-format-plugin @drsooch
+/plugins/hls-brittany-plugin @fendor
+/plugins/hls-call-hierarchy-plugin @July541
+/plugins/hls-class-plugin @Ailrun
+/plugins/hls-eval-plugin @pepeiborra
```

I handed over the eval plugin to @tittoassini (https://github.com/haskell/haskell-language-server/pull/438#pullrequestreview-543322357)

```suggestion
/plugins/hls-eval-plugin @tittoassini
```